pallets/runtime/develop/src/runtime.rs

```diff
@@ -374,16 +370,18 @@ impl pallet_staking::Trait for Runtime {
 parameter_types! {
     pub const MotionDuration: BlockNumber = 0;
 }
+
+/// Voting majority origin for `Instance`.
+type VMO<Instance> = committee::EnsureProportionAtLeast<_1, _2, AccountId, Instance>;
```

Comment:
Let's use PascalCase for types. Either avoid acronyms or write them in PascalCase as well. The front end lib we use (polkadot.js) automatically converts PascalCase and snake_case to camelCase. This means, `VMO` becomes `vMO` and it looks weird :)

Reply:
Per discussion on slack: just a private type alias & VMO is PascalCase cause its an initialism.